plugins/pulp_rpm/plugins/importers/yum/upload.py

```diff
@@ -179,42 +181,56 @@ def _handle_erratum(repo, type_id, unit_key, metadata, file_path, conduit, confi
     unit.save()
 
     if not config.get_boolean(CONFIG_SKIP_ERRATUM_LINK):
+        rpm_filter_list = []
+        for search_dict in unit.rpm_search_dicts:
+            # TODO should we del checksum and checksumtype from search_dict
+            rpm_filter_list.append(Q(**search_dict))
+        Q_rpm_filter = reduce(operator.or_, rpm_filter_list)
         for model_type in [models.RPM, models.SRPM]:
-            pass  # TODO Find out if the unit exists, if it does, associated, if not, create
+            for unit in model_type.objects.filter(Q_rpm_filter):
+                # TODO associate the unit w/ this errata
```

Comment:
I don't know how to fix this. It originally was implemented with `conduit.link_unit()` which also appears to be hard coded for the old database layout (`_id` becoming `repo_id` and the like).

What do I need to do to create a 'bi-directional' link between rpm/srpms and erratum?